week 5/README.md

```diff
@@ -109,6 +109,47 @@ for (int i = 0; i < n; ++i)
 
 ### Arrays
 
+The **array** is such a **data structure** that stores a fixed-size sequencial elements of a certain type.
+All elements of the array occupy contiguous memory locations. The lowest address corresponds to the first element and the highest address to the last element.
```

Comment:
Ok, we'll leave this for pointers. Do you think we should add something else about arrays?